ambient-light/AmbientLightSensor_stop.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>AmbientLightSensor Test: stop()</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://www.w3.org/TR/ambient-light/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+
+var sensor, stop_return;
+setup(function() {
+  sensor = new AmbientLightSensor();
+  sensor.start();
+  stop_return = sensor.stop();
+});
+
+test(function() {
+  assert_equals(sensor.state, "idle");
```

Comment:
I think it is better to put all these assertions in this test file together within `ambient-light/AmbientLightSensor_onstatechange.html`.